table.csv

```diff
@@ -70,6 +70,7 @@ ipfs-ns,                        namespace,      0xe3,           IPFS path
 swarm-ns,                       namespace,      0xe4,           Swarm path
 ipns-ns,                        namespace,      0xe5,           IPNS path
 ed25519-pub,                    key,            0xed,           Ed25519 public key
+ed25519-sig,                    signature,      0xef,           Ed25519 signature
```

Comment:
Why not make `ed25519-{pub,sig}` mean using Ed25519-SHA2 (since that's what effectively all Ed25519 users use), and then add only `ed25519-{pub,sig}-sha3`? It seems a bit odd to me to have two codec entries which are ambiguous.